components/alert/alert.css

```diff
@@ -1,82 +1,68 @@
-/* || IMPORTS */
-@import url("../../utilities/global.css");
-@import url("../../helper/spacing.css");
-@import url("../../helper/size.css");
-@import url("../../helper/flex.css");
-@import url("../../helper/doc.css");
-
 /* || ROOT CSS */
 :root {
-    --primary-alert-light: #CFE2FF;
-    --secondary-alert-light: #E2E3E5;
-    --success-alert-light: #D1E7DD;
-    --danger-alert-light: #F8D7DA;
-    --warning-alert-light: #FFF3CD;
-    --info-alert-light: #CFF4FC;
-    --primary-alert-dark: #2864be;
-    --secondary-alert-dark: #808080;
-    --success-alert-dark: #17925a;
-    --danger-alert-dark: #ca2b39;
-    --warning-alert-dark: #b89628;
-    --info-alert-dark: #2d96ad;
-}
-
-/* || COMPONENT CSS */
-.component-type:not(:last-child) {
-  border-bottom: 3px solid var(--alert-color);
-}
-.active-component {
-  border-radius: 0.3rem;
-  background-color: var(--alert-color);
+  --primary-alert-light: #cfe2ff;
+  --secondary-alert-light: #e2e3e5;
+  --success-alert-light: #d1e7dd;
+  --danger-alert-light: #f8d7da;
+  --warning-alert-light: #fff3cd;
+  --info-alert-light: #cff4fc;
+  --primary-alert-dark: #2864be;
+  --secondary-alert-dark: #808080;
+  --success-alert-dark: #17925a;
+  --danger-alert-dark: #ca2b39;
+  --warning-alert-dark: #b89628;
+  --info-alert-dark: #2d96ad;
```

Comment:
Avoid making super-specific CSS variables. You may reuse primary, success, and warning colors in other places.

Reply:
But i wanted a different shade of it. That is why.